src/card.css

```diff
@@ -0,0 +1,103 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+}
+
+.imageCard {
+  display: block;
+  margin: 32px auto;
+  width: 160px;
+  height: 134px;
+}
+
+.infoAboutCard {
+  margin: 0 16px;
+}
+
+.title {
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  font-family: Roboto, sans-serif;
+  margin-bottom: 4px;
+}
+
+.code {
+  font-family: Roboto-Regular, sans-serif;
```

Comment:
you don't need to write `font-family` in each class, it's better to write it in `html` tags - then the font will be applied to the page like this:

```
html {
  font-family: "Roboto", sans-serif;
  font-size: 10px;
  font-weight: 400;
  line-height: 14px;
}
```

Reply:
l just used different fonts, so l wrote it like this